package.json

```diff
@@ -74,27 +74,27 @@
     "@storybook/addon-knobs": "^3.4.11",
     "@storybook/addon-links": "^3.4.11",
     "@storybook/react": "^3.4.11",
-    "autoprefixer": "^9.1.5",
+    "autoprefixer": "^9.4.3",
```

Comment:
Interestingly enough one of the [changes](https://github.com/postcss/autoprefixer/blob/master/CHANGELOG.md) under the hood is a migration to babel 7 - perhaps this is just in their build process though.